### PR TITLE
Fix design issues

### LINKED ISSUE #288: Changes/Improvements to Onboarding
### What problem does this address?

Based on the feedback received, here are things that we need to address in the Onboarding wizard. It should be noted that we can estimate this issue and if the estimate is larger than medium we can split this into multiple smaller issues and proceed.

### What is your proposed solution?

- [x] **Top Header Issue:** The top header of the onboarding process interferes with the starter site preview. It gives the impression that the white bar is part of the starter site itself.

Please refer to this comment for the changed design: https://github.com/Codeinwp/templates-patterns-collection/issues/288#issuecomment-1735176864

Screenshot 2023-09-25 at 10 01 24 PM

---

- [x] **Next Step Button:** On full screen, the ‘Next Step’ button is not visible without scrolling down. This could frustrate users and increase the drop rate.
Screenshot 2023-09-25 at 10 05 39 PM (2)

We can ensure that the button always stays in the view (floating on the bottom), and is inclusive of all the screens with similar buttons.

---

- [x] **Missing UTM Tags:** We can revisit the code to confirm no external links are missing UTM tags, an example [here](https://github.com/Codeinwp/templates-patterns-collection/blob/6d79cde9abb184f4e0e4270e5617dd5b6d43f330/onboarding/src/Components/SiteSettings.js#L201).

---

- [x] **Textarea in Logo Section:** The textarea in the logo section doesn’t make sense as it doesn’t allow URL input. It only activates with the ‘Browse’ button. Also reported at https://github.com/Codeinwp/neve-pro-addon/issues/2634

Please refer to this comment for the changed design: https://github.com/Codeinwp/templates-patterns-collection/issues/288#issuecomment-1735176864

Annotation-Annotation on 2023-09-25 at 18-46-38 png

---

- [x] **Missing Shadow/Border:** During the preview, the sidebar doesn't have any shadow/border, which doesn't look right. Similar to what we have in the header, we can have some shadow or border here to make the hierarchy clear as it stands in the Figma mockup.

Screenshot 2023-09-25 at 10 11 13 PM

---

- [x] **Thumbnail Size in Larger Display:** The thumbnail ratio in larger displays isn't ideal and can be improved to look more natural than it currently is, or we could display more sites there to avoid it looking stretched.

Screenshot 2023-09-25 at 10 11 06 PM

### Will this feature require documentation? (Optional)

None

### PR DESCRIPTION
**Pull request by cristian-ungureanu**
### Summary
In this PR I address the following issues:
- Remove the top header for certain steps
- Make the next button sticky
- Added missing UTM tags
- Change the look of the logo component
- Added missing box shadow for the settings sidebar
- Keep the same aspect ratio for starter site cards

Note: After discussing this with Hardeep, the loading conditions for Onboarding were changed. The onboarding doesn't check for Neve's new users anymore. In order to load, here's what it checks for:
- User can install plugins
- Starter sites are not disabled in white-label module and
- The current theme is Neve or a child theme of it and
- It's the first installation of TPC and
- The onboarding process was not finished

### Test instructions
- Install and activate Neve
- Install this version of TPC and activate it
- The onboarding should start as soon as you activate TPC
- Check and see if you can find any issues with the implementation of the previously mentioned changes

Closes #288.

### CODE REVIEW

File: templates-patterns-collection.php [hunk 1]
```diff
@@ -60,6 +60,10 @@ function ti_tpc_load_textdomain() {
 define( 'TIOB_PATH', dirname( __FILE__ ) . '/' );
 define( 'TIOB_BASENAME', plugin_basename( __FILE__ ) );
 
+if ( empty( get_option( 'templates_patterns_collection_install' ) ) ) {
+	// If the installation time is not set, it means TPC is installed for the first time.
+	update_option( 'tpc_obd_new_user', 'yes' );
+}
```
Comment (HardeepAsrani) [inline templates-patterns-collection.php:67]: We should be able to move it to `ti_tpc_activation_redirect` function.

**Review by HardeepAsrani — APPROVED** [review]
Looks good, left a small feedback. We can perhaps also get design team to review the changes to confirm they don't find anything miss in parallel to QA.

**Comment by cristian-ungureanu** [comment]
Hey @Codeinwp/design-team, can you have a look here and see if there are any more required changes? 🙏

**Comment by cristian-ungureanu** [comment]
Thanks, @JohnPixle! The opacity was not really an option as it would have applied to the text of the button.
When I raised the issue I was a bit fooled since I've tested the onboarding on WP 5.5 ( had another task that required that version and forgot to update ) 
Having the dark color as a hover is a better option and can be implemented. Thank you for having another look over it 🚀

**Comment by mghenciu** [comment]
Super nice work, Cristi 🔝 🚀 

Just some things, that may be caused by my browser or intended, but I'll mention them:

- [ ] when selecting Font Families, for me the selection outline disappears (same for hover). I think it should stay on the selected Box - even when selected. Basically use the same hover and select principles as the preview on demosites.io

https://github.com/Codeinwp/templates-patterns-collection/assets/52494172/cbb78948-9df7-43bc-aca4-692a00995782

___

- [ ] the builder dropdown, ideally should close itself when clicking outside.
- [ ] maybe we shouldn't 'scream' in the penultimate screen by using exclamation sign. Use something like this (below). But it's ok to use the exclamation on the last screen, when congratulating the user. 
image


Thank you!

**Review by HardeepAsrani — APPROVED** [review]

**Comment by rodica-andronache** [comment]
hey @cristian-ungureanu https://demosites.io/ seems to be down now, so I need to wait for that before testing https://vertis.d.pr/i/iocG9V.

**Comment by rodica-andronache** [comment]
@cristian-ungureanu I've tested on a normal and multisite, with various users. Found some issues on multisite, but they are not related to this PR, same thing happens for the current version as well: https://github.com/Codeinwp/templates-patterns-collection/issues/292. Not sure if they should be addressed now or not.

Regarding the current PR:

- [ ] `Maybe when we are in the onboarding, we can disable the Color Palette Switcher from each individual starter site, as now it’s not easy to understand where you can actually change the colours and fonts for the preview https://vertis.d.pr/i/rwQzKi ?` -> this thing that was previously mentioned is still happening. When you switch colors from the onboarding, the Color Palette Switcher from each individual starter site disappears, but reappears if you change the page https://vertis.d.pr/v/LrfZYQ


- [ ] A question: While in the onboarding window, when clicking on links with `target=“_blank”` from a starter site, you are being taken out of the onboarding window. I know that’s normal for a link with that atribute. I was just wandering if this is something that we should consider somehow. ( Like have something done from the onboarding for it to not work like that, or maybe update the starter sites to not have any external links, or not do anything about it )? https://vertis.d.pr/v/z2c3Ue

**Comment by cristian-ungureanu** [comment]
Hey @rodica-andronache, I managed to fix the first raised issue.

The second one would require some changes in the demo data exporter repo. I think we should handle it separately as it's not that important and we can live with it for now. We can open a new issue for this and @HardeepAsrani will prioritize it as he sees fit.

About the multisite issues you've mentioned, I think they are the same as you've mentioned in this PR, right? If yes, we can handle them there.

Let me know your thoughts.

**Review by HardeepAsrani — APPROVED** [review]

**Comment by rodica-andronache** [comment]
@cristian-ungureanu I've tested the first issue and is fixed now. For the second one, I created a new issue [here](https://github.com/Codeinwp/demo-data-exporter/issues/190).

The issues I've mentioned https://github.com/Codeinwp/templates-patterns-collection/issues/292 do exist in this PR and in the current version of TPC

**Comment by cristian-ungureanu** [comment]
Thank you for your awesome testing, Rodica. 🚀 

If you agree, we can move this to "Ready to merge" and we'll handle everything else separately.

**Comment by rodica-andronache** [comment]
Hey @cristian-ungureanu if you feel it's easier we can move this to ready to merge, and fix everything separately, but we'll have to test it again, before actually releasing this, because I think there are still issues.

As I mentioned in the latest comment here https://github.com/Codeinwp/neve/pull/4108 that issue is still not fixed with the latest commit from this PR.
Also, with that last commit on this PR, with the current version of Neve, there's another issue happening: On a multisite, when logged in as administrator ( not super admin ), with TPC installed and activated, if you activate Neve, you get "Sorry you are not allowed to access this page" https://vertis.d.pr/v/Gyg6jh

**Pull request merged**